starcheck/src/starcheck.pl

```diff
@@ -397,7 +422,7 @@
 
 foreach my $oflsid (keys %guidesumm){
     unless (defined $obs{$oflsid}){
-	warning("OFLS ID $oflsid in Guide Summ but not in DOT! \n");
+     #warning("OFLS ID $oflsid in Guide Summ but not in DOT! \n");
```

Comment:
Logic change.

Reply:
Yes revert that. The new feature to limit the number of obsids during a development cycle doesn't play well with this check and generates tons of warnings.

Reply:
Seems like we should keep the check then but skip it if MAX_OBSIDS is set?

Reply:
Sounds good.